### DISCUSSION TITLE
ILI9325 driver support

### DISCUSSION THREAD

**Discussion by LeonRSA** [Bodmer/TFT_eSPI#959] [New processor/display support or capability request]
Good day Bodmer,

Does this library support the ILI9325 driver?
If not, can a different driver be used, eg. ILI9341. etc?

Regards,
Leon

**Comment by Bodmer** [comment]
The library has no embedded support for the ILI9325 so some experimentation will be required.

The library is designed to be able to add new displays but I can only add them when I have the particular display myself.

**Reply by TangerineDreamer** [reply]
Hi,
why not checking here:
https://github.com/Nkawu/TFT_22_ILI9225

they also talk  about a similar tft but with a 9341 driver.

**Reply by Bodmer** [reply]
The library already supports the ILI9225, the OP asks about the ILI9325
